h/static/scripts/directives/test/annotation-test.coffee

```diff
@@ -97,30 +97,6 @@ describe 'h.directives.annotation', ->
   afterEach ->
     sandbox.restore()
 
-  describe 'when the annotation is a highlight', ->
```

Comment:
We want these tests. You should restore them. The thing that needs to change is line 102.